data_internet.go

```diff
@@ -2,6 +2,8 @@ package gofakeit
 
 // DataInternet consists of various internet information
 var DataInternet = map[string][]string{
-	"browser":       {"firefox", "chrome", "internetExplorer", "opera", "safari"},
-	"domain_suffix": {"com", "biz", "info", "name", "net", "org"},
+	"browser":           {"firefox", "chrome", "internetExplorer", "opera", "safari"},
+	"domain_suffix":     {"com", "biz", "info", "name", "net", "org"},
```

Comment:
I think we can come up with a few more. I know "edu", "me", "io" are just  a few, not looking for all of them but a few more popular ones would be nice.

Reply:
Haha i just realized this was mine. if you could make this change I would be appreciated. Otherwise i can do it myself.